suppression-backup-service/cmd/main.go

```diff
@@ -0,0 +1,139 @@
+package main
```

Comment:
We are missing docker-related adaptations. With respect to that, I believe that we should start producing separate docker images for this and other similar processes, i.e. have a separate `Dockerfile` & push separate docker images. E.g. regulation-worker should have its own image too (in the future as a separate task).

Piggybacking on rudder-server's docker image won't get us too far.

Reply:
make sense. I have added a ticket for the same on notion board.

Reply:
Shouldn't we add the docker image for this new command as part of this task?

Reply:
okay, I thought you are suggesting to create a `Dockerfile` for both `regulation-worker` & `suppression-backup-server` both in separate ticket. Anyways, I will add the `Dockerfile` for this service in this PR only.

Reply:
We'll need a github action for building the image too

Reply:
yup